Loop/Managers/LoopDataManager.swift

```diff
@@ -1647,7 +1649,9 @@ extension LoopDataManager {
                     lastTempBasal: lastTempBasal,
                     volumeRounder: volumeRounder,
                     rateRounder: rateRounder,
-                    isBasalRateScheduleOverrideActive: settings.scheduleOverride?.isBasalRateScheduleOverriden(at: startDate) == true
+                    isBasalRateScheduleOverrideActive: settings.scheduleOverride?.isBasalRateScheduleOverriden(at: startDate) == true,
+                    insulinOnBoard: self.insulinOnBoard?.value,
```

Comment:
If we have a failure to get insulin on board, then dosing will not be limited. The `insulinOnBoard` function parameter here should probably be non-optional and the loop fail with an error if it can't be retrieved.

Reply:
I naively thought the code where self.insulinOnBoard is set:

https://github.com/marionbarker/Loop/blob/wip/max-auto-iob/Loop/Managers/LoopDataManager.swift#L1039-L1048

with the switch for result of .failure or .success handles the case where there is a failure to get insulin on board.

Reply:
Looking more closely (and finding your earlier comment) - it does seem to be an a warning.
Changing this is beyond what I can do.